A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -547,9 +547,12 @@ class FMCMainDisplay extends BaseAirliners {
                     this.showErrorMessage("ENTRY OUT OF RANGE");
                     return false;
                 }
+            } else {
+                this.showErrorMessage("FORMAT ERROR");
+                return false;
             }
         }
-        this.showErrorMessage(this.defaultInputErrorMessage);
+        this.showErrorMessage("FORMAT ERROR");
```

Comment:
Maybe we should add those FORMAT ERROR strings as a class field like `this.defaultInputErrorMessage`.

Reply:
Sure! Will give it a go when i get some time